src/index.js

```diff
@@ -2,34 +2,38 @@ import './style.css';
 import morevert from './images/more-vert.svg';
 import refresh from './images/autorenew.svg';
 import add from './images/add.svg';
+import Interact from './module/display.js';
+import { getTodos } from './module/storage.js';
+import addTask from './module/addTask.js';
 
+const todos = getTodos();
+const interact = new Interact(todos);
 const todaysTodo = document.querySelector('.todays-todo');
 const addTodo = document.querySelector('.add-todo');
-const todoList = document.querySelector('.todo-list');
 const completeAll = document.querySelector('.completeAll');
 
-const todos = [{ description: 'Wash clothes', completed: false, index: 1 },
-  { description: 'Make dinner', completed: false, index: 2 },
-  { description: 'wash dishes', completed: false, index: 3 },
-  { description: 'clean house', completed: false, index: 4 }];
+function render() {
+  todaysTodo.innerHTML = `<h4 class="title">Today's To do</h4>
+  <button class="refreshBtn"><img src="${refresh}" alt=""></button>`;
+  addTodo.innerHTML = `<input type="text" class="input-todo required" placeholder="Add to your list...">
+  <button class="addBtn"><img src="${add}" alt=""></button>`;
 
-todaysTodo.innerHTML = `<h4 class="title">Today's To do</h4>
-<button class="refreshBtn"><img src="${refresh}" alt=""></button>`;
+  completeAll.innerHTML = `<input class="check" type="checkbox">
+  <span class="todo-title bold">Complete To Do list project</span>
+  <button class="more-btn"><img src="${morevert}" alt="more-vertical-icon"></button>`;
+}
 
-addTodo.innerHTML = `<input type="text" class="input-todo" placeholder="Add to your list...">
-<button class="addBtn"><img src="${add}" alt=""></button>`;
+render();
 
-todos.forEach((todo) => {
-  const list = document.createElement('li');
-  list.className = 'list';
-  list.innerHTML = `
-  <input id="${todo.index}" class="check" type="checkbox">
-  <span class="todo-title">${todo.description}</span>
-  <button id="${todo.index}" class="more-btn"><img src="${morevert}" alt="more-vertical-icon"></button>
-  `;
-  todoList.appendChild(list);
-});
+interact.display();
+
+const inputTodo = document.querySelector('.input-todo');
+const addBtn = document.querySelector('.addBtn');
 
-completeAll.innerHTML = `<input class="check" type="checkbox">
-<span class="todo-title bold">Complete To Do list project</span>
-<button class="more-btn"><img src="${morevert}" alt="more-vertical-icon"></button>`;
+addBtn.addEventListener('click', () => {
```

Comment:
[OPTIONAL] Good job with adding and removing functionality. But I highly recommend to use enter key to add a new to-do to the list. In this way, you will increase your Web app's UX.